### PR TITLE
Add troubleshooting advice about the --userns option. [CI:DOCS]

### PR DESCRIPTION
**Pull request by jberkus**
While I'm at it, add a link to the troubleshooting guide into
the issue template.

Thanks @mheon for the info about userns

### CODE REVIEW

**Comment by mheon** [comment]
Gitvalidation is complaining about your commit - looks like you're missing a signoff. Can you do a `git commit --amend -s` to add one and re-push?

Also, you can add `[CI:DOCS]` to the title of your PR to avoid most of our CI checks, as this is a docs-only change.

File: troubleshooting.md [hunk 1]
```diff
@@ -47,6 +47,14 @@ will disable SELinux separation for the container.
 
 $ podman run --security-opt label=disable -v ~:/home/user fedora touch /home/user/file
 
+In cases where the container image runs as a specific, non-root user, though, the
+solution is to fix the user namepsace.  This would include container images such as
```
Comment (eriksjolund) [inline troubleshooting.md:51]: namepsace -> namespace

**Comment by rhatdan** [comment]
@jberkus You need to sign your PRs.

git commit -a --amend -s
git push --force

**Comment by jberkus** [comment]
PR updated.

**Comment by rhatdan** [comment]
Josh, one more change, you need to put  a blank line in between your title line and the first line describing the fix.  For some reason the tests see these as the same line, unless there is a blank line in between.

**Comment by jberkus** [comment]
Dan: that sounds like a bug in the tests.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -67,7 +67,8 @@ Briefly describe the problem you are having in a few paragraphs.
 (paste your output here)
 ```
 
-**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide?**
+**Have you tested with the latest version of Podman and have you checked the Podman Troubleshooting Guide? (https://github.com/containers/podman/blob/master/troubleshooting.md)**
+
```
Comment (TomSweeneyRedHat) [inline .github/ISSUE_TEMPLATE.md:71]: nice change, I hadn't thought to add a link like this in a template....

**Comment by TomSweeneyRedHat** [comment]
LGTM
once tests are hip
Thanks @jberkus !

**Pull request closed without merging**